test/integration/02-user-wallet/02-receive-lightning.spec.ts

```diff
@@ -2,6 +2,8 @@ import { ledger } from "@services/mongodb"
 import { getHash } from "@core/utils"
 import { checkIsBalanced, getUserWallet, lndOutside1, pay } from "test/helpers"
 import { MEMO_SHARING_SATS_THRESHOLD } from "@config/app"
+import * as Wallets from "@app/wallets"
```

Comment:
the module should be `wallet` (singular)

Reply:
I've consistently been using pluralisation for all module names
```
src/domain
├── accounts
├── bitcoin
│   ├── lightning
│   └── onchain
├── ledger
├── primitives
├── users
├── wallet-invoices
└── wallets
```